Sources/Starknet/Data/Responses.swift

```diff
@@ -39,3 +39,31 @@ public struct StarknetBlockHashAndNumber: Decodable, Equatable {
         case blockNumber = "block_number"
     }
 }
+
+public struct GetEventsResponse: Decodable, Equatable {
```

Comment:
Let's keep the naming convention and start all names that are public with Starknet. (Whether this is a good convention is to be discussed)